pom.xml

```diff
@@ -354,6 +354,55 @@
         <argLine>-Xms4g -Xmx4g -Dlog4j2.configurationFile=log4j2.xml</argLine>
       </properties>
     </profile>
+    <profile>
+      <id>codecoverage</id>
+      <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <groupId>org.jacoco</groupId>
+              <artifactId>jacoco-maven-plugin</artifactId>
+              <executions>
+                <execution>
+                  <!-- used when runnning jacoco:report-aggregate@report via commandline -->
+                  <id>report</id>
+                  <configuration>
+                    <skip>true</skip> <!-- default for all modules, overwrite for root project -->
+                  </configuration>
+                </execution>
+              </executions>
+            </plugin>
+          </plugins>
+        </pluginManagement>
+        <plugins>
+          <plugin>
+            <groupId>org.jacoco</groupId>
+            <artifactId>jacoco-maven-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>prepare-agent</id>
+                <goals>
+                  <goal>prepare-agent</goal>
+                </goals>
+                <configuration>
+                  <append>true</append>
+                  <includes>
+                    <include>org/apache/pinot/**/*</include>
+                  </includes>
+                </configuration>
+              </execution>
+              <execution>
+                <id>report</id>
+                <inherited>false</inherited>
+                <configuration>
+                  <skip>false</skip>
```

Comment:
I don't fully follow this. If we override it here, do we need to declare it in the plugin management?